src/VocabularyManager.Core/Services/WordListService.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Ardalis.Specification;
+using VocabularyManager.Core.Entities;
+using VocabularyManager.Core.Exceptions;
+using VocabularyManager.Core.Interfaces;
+using VocabularyManager.Core.Specifications;
+
+namespace VocabularyManager.Core.Services
```

Comment:
why implementation of interface it's in core?

Reply:
I saw almost the same at eShop [project](https://github.com/dotnet-architecture/eShopOnWeb/tree/main/src/ApplicationCore/Services)

Reply:
:( 

https://github.com/jasontaylordev/CleanArchitecture

this is better example